src/components/Header.test.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import Header from "./Header";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe("When the Header Load...", () => {
+  let header;
+  beforeEach(() => {
+    header = shallow(<Header />);
+  });
+
+  it("displays a navlink on the header", () => {
+    //Arrange the Header
+
+    //Act on the header by finding an element
+    const navLinkWrapper = header.find("NavLink");
+    expect(navLinkWrapper.length).toEqual(3);
+  });
+  
+  
+  it("checks for a tag by its class name to contain some text", () => {
```

Comment:
This "it" statement is a bit off for what this test is doing. It's alright to leave this check mark undone if you don't have a class to call upon.

Reply:
Fixed this it statement.